### PR TITLE
basic design- authpages, notes page, theme customisation , material ui , not functionality implemented yet..

### PR DESCRIPTION
**Pull request by kgajriaAgarwal**
Hello, i have Addedbasic design- authpages, notes page, theme customisation , material ui , not functionality implemented yet. kindly review it and let me know what do think about it !
PREVIEW - https://blithe-la-nota.netlify.app
files to check- Auth folder, context, mainpage folder

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
+        default: "#202124"
+      },
+      primary: {
+        main: '#A14F57',
+      },
+      secondary: {
+        main: '#ffffff',
+      },
+      text: {
+        primary: "#ffffff",
```
Comment (MayankKumar10) [inline src/App.js:48]: Suggestion: try to define color in root directory

File: src/Pages/Auth/Auth.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    width: 22rem;
+}
+
+.divider{
+    color: var(--primary-bg);
+    font-weight: 800;
+}
+
+.auth-form{
+    display: flex;
```
Comment (MayankKumar10) [inline src/Pages/Auth/Auth.css:18]: Suggestion: If you use the flexbox center elsewhere, do make a helper class then.

File: src/Pages/Auth/Login.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    FormControl,
+    InputLabel,
+    OutlinedInput,
+  } from "@mui/material";
+  import react, { useState } from "react";
+  import { Link } from "react-router-dom";
+  import bgImg from "../../Assets/Images/bg-collage.jpg";
+  import blitheLogo from "../../Assets/logo/logo-blue.jpg";
+  import { useThemeMode } from "../../Helpers/Context";
+  // import { SwipeableStepper } from '../Shared';
```
Comment (MayankKumar10) [inline src/Pages/Auth/Login.js]: Suggestion: remove this commented code

File: src/Pages/Auth/Login.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+        return {
+          ...prevState,
+          show_password: prevState.show_password==='true' ? "false" : "true",
+        };
+      });
+    };
+  
+    const handleLogin = (e) => {
+      e.preventDefault();
+      // console.log("handle login..");
```
Comment (MayankKumar10) [inline src/Pages/Auth/Login.js]: Suggestion: remove this commented lines also

File: src/Pages/MainPage/RouteData.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        </p>
+      </div>
+    );
+  }
+
+//Routing Data
+const RouteData = () =>{
+    return(
+        <Routes>  
+           {/* <Route exact path='/user' element={<PrivateRoute/>}>
```
Comment (MayankKumar10) [inline src/Pages/MainPage/RouteData.js:29]: Suggestion: Remove this commented code.

**Review by MayankKumar10 — APPROVED** [review]
Nice UI, Everything Looks Good Just Look into these Suggestions, you can merge it

**Review by thenorthkun — APPROVED** [review]
There's a lot of commented code across different files which can be removed. Apart from that the rest of the code looks fine. You can make the notes more customimzable & a bit more oreiented to note taking,  Nice that you added dark mode feature along as well.

File: src/App.js (hunk 1 above)
Comment (kgajriaAgarwal) [reply src/App.js:48]: ok this of theme

File: src/Pages/Auth/Auth.css (hunk 2 above)
Comment (kgajriaAgarwal) [reply src/Pages/Auth/Auth.css:18]: ok

File: src/Pages/Auth/Login.js (hunk 4 above)
Comment (kgajriaAgarwal) [reply src/Pages/Auth/Login.js]: ok

**Pull request merged**